packages/rum-core/src/common/utils.js

```diff
@@ -200,13 +200,34 @@ function getTimeOrigin() {
   return PERF.timing.fetchStart
 }
 
-function getPageMetadata() {
+function getNetworkInformation() {
+  const connection = navigator && navigator.connection
+
+  if (!(connection && typeof connection === 'object')) {
+    return
+  }
+  /**
+   * Ignoring `type` and `downlinkMax` as they are only
+   * supported on Chrome OS
+   */
   return {
-    page: {
-      referer: document.referrer,
-      url: window.location.href
-    }
+    downlink: connection.downlink,
+    effective_type: connection.effectiveType,
+    rtt: connection.rtt,
+    save_data: !!connection.saveData
+  }
```

Comment:
Do you plan to define these fields in apm server schema? if so maybe reference the PR.

Is there any reason not to include `connection.type`, it includes things like `cellular`, `wifi`, etc which is useful in categorising the network.

Reply:
Added the link to server schema. `type` is available only on chromeOS and its an exception.